From: Michal Marek <mmarek@suse.cz>
Subject: Fix kabi signature of struct jbd2_buffer_trigger_type
Patch-mainline: never

patches.fixes/jbd2-ocfs2-Fix-block-checksumming-when-a-buffer-is-u.patch
renames the callback for the sake of clarity, the bits remain the same. Hide
this from genksyms.

Signed-off-by: Michal Marek <mmarek@suse.cz>
---
 include/linux/jbd2.h |    7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

--- linux-2.6.32-SLE11-SP1.orig/include/linux/jbd2.h
+++ linux-2.6.32-SLE11-SP1/include/linux/jbd2.h
@@ -1047,7 +1047,12 @@ struct jbd2_buffer_trigger_type {
 	 * before a buffer is written to the journal.  mapped_data is a mapped
 	 * buffer that is the frozen data for commit.
 	 */
-	void (*t_frozen)(struct jbd2_buffer_trigger_type *type,
+#ifndef __GENKSYMS__
+	void (*t_frozen)
+#else
+	void (*t_commit)
+#endif
+		(struct jbd2_buffer_trigger_type *type,
 			 struct buffer_head *bh, void *mapped_data,
 			 size_t size);
 
